core/arch/arm/kernel/generic_entry_a64.S

```diff
@@ -423,7 +423,6 @@ END_FUNC unhandled_cpu
 	  .endif
 	.endm
 
-	.section .identity_map
 	.align	11
```

Comment:
What happens with this, will it still work as expected?

Reply:
Oh cr*p, no it doesn't :cry: `reset_vect_table` is not aligned on `2**11` bits anymore!

I see two options (both tested, alignment looks OK in both cases):

1. Just append expected flags and type: `.section .identity_map, "ax", %progbits`.
2. Do not use `LOCAL_FUNC`:
  ```
           .section .identity_map, "ax", %progbits
           .type reset_vect_table, %function
           .align 11
  reset_vect_table:
            etc.
  ```

What do you think?

Reply:
Of those two I'd prefer option 1.

Perhaps we could add an optional alignment argument to `FUNC()` and `LOCAL_FUNC()`?

Reply:
Section name is already optional so it is not very convenient to add an optional alignment. Since we have only one use case anyway, it seems more reasonable to just keep our previous approach and just make it correct by adding the flags and type.

I will to that and slightly rework the commit text.